managed/ActivityTypeFundingApplicationTaskInternal.mgd.php

```diff
@@ -15,8 +17,8 @@
       'values' => [
         'option_group_id.name' => 'activity_type',
         'label' => E::ts('Internal Funding Application Task'),
-        'value' => \Civi\Funding\ActivityTypeIds::FUNDING_APPLICATION_TASK_INTERNAL,
-        'name' => 'funding_application_task_internal',
+        'value' => ActivityTypeIds::FUNDING_APPLICATION_TASK_INTERNAL,
+        'name' => ActivityTypeNames::FUNDING_APPLICATION_TASK_INTERNAL,
```

Comment:
Would it make sense to somehow merge `ActivityTypeIds` and `ActivityTypeNames`?

Reply:
As Civi is now able to determine `value`, `ActivityTypeIds` should be completely dropped. Though I'll leave that for a later refactoring.